src/Command/JobCommand.php

```diff
@@ -91,7 +91,7 @@ public function execute(Arguments $args, ConsoleIo $io) {
 			}
 
 			foreach ($jobs as $job) {
-				$io->out('- [' . $job->id . '] ' . $job->job_task . ' (' . ($job->completed ? 'completed' : 'failed ' . $job->failed . 'x') . ')');
+				$io->out('- [' . $job->id . '] ' . $job->job_task . ' (' . ($job->completed ? 'completed' : 'failed ' . $job->attempts . 'x') . ')');
```

Comment:
Shouldn't this string `failed` also be renamed to `attempts`?

Reply:
Well but it failed with x attempts

Reply:
right

Reply:
Updated 👍 Though I don't think it really matters, because if the job is _not_ completed, the "attempts" field would represent number of "failures" anyway. But yeah, I have confused myself around these words so much already while doing this lol